source/routing/isochrone.cpp

```diff
@@ -258,7 +258,7 @@ std::vector<Isochrone> build_isochrones(RAPTOR& raptor,
             } else {
                 output = max_isochrone;
             }
-            isochrone.push_back(Isochrone(output, duration[i], duration[i-1]));
+            isochrone.insert(isochrone.begin(), Isochrone(std::move(output), duration[i], duration[i-1]));
```

Comment:
Inserting at the beginning of a vector in a loop is a bad habit. inserting at the beginning of a vector in O(n), doing that n times in a loop make it O(n²). Better to push_back() and then reverse the vector.